src/components/app/navigation.styles.scss

```
@@ -685,3 +692,274 @@ $unreadBubbleAnimation: count 500ms ease 500ms forwards;
+		display: flex;
+		align-items: center;
+		justify-content: center;
+	}
+
+	.navigation__icon__single,
+	.navigation__globe-svg,
+	.navigation__liveChatToggleIcon {
+		width: 32px !important;
+		height: 32px !important;
+		display: block;
+		color: currentColor;
```

Stylelint (17.13.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)